src/main/resources/logback-spring.xml

```diff
@@ -39,8 +39,21 @@
         <appender-ref ref="aiAppender"/>
     </logger>
 
+    <logger name="com.microsoft.applicationinsights" additivity="false" level="INFO">
```

Comment:
I take it this was just for debugging what was going wrong at the time - is it still needed or is it best to decrease to `WARN`?

Reply:
It was part of eliminating differences - now removed and checked that events are still generated. The core issue was in the way the key was being set.